images/ubuntu/scripts/build/configure-apt.sh

```diff
@@ -47,3 +47,4 @@ if ! is_ubuntu24; then
     # https://github.com/ilikenwf/apt-fast
     bash -c "$(curl -fsSL https://raw.githubusercontent.com/ilikenwf/apt-fast/master/quick-install.sh)"
 fi
```

Comment:
@vidyasagarnimmagaddi why do you need to move this file?

Reply:
Please check issue `#9780`, which states why it broke and the debugging steps we took.

Reply:
the investigation there does not seem to be correct `which` mentioned in the issue works fine for both `/usr/local/bin` and /usr/local/sbin` (I have just checked and it works fine)

Reply:
@bryan-bar both `/usr/local/bin` and `/usr/local/sbin` are in $PATH, for software report we are using the `which apt-fast` command, `which` in turn scan everything it finds in `$PATH` you can see my runtime execution in the screenshot (silly step names if you do not mind, just a debug.

![image](https://github.com/actions/runner-images/assets/88318005/a1f5b8ff-6783-4499-a352-b4aeb97c12b3)


1. line 7 - I check `which apt-fast`
2. I remove current apt-fast installation from the current image
3. I reinstall it in runtime
4. I check `which apt-fast` again, it is now in the new path
5. I run the

```
function Get-AptFastVersion {
    $versionFileContent = Get-Content (which apt-fast) -Raw
    $match = [Regex]::Match($versionFileContent, '# apt-fast v(.+)\n')
    return $match.Groups[1].Value
}
```
function, which is used in the software generator and get the version for the newly installed path, my take is that `/usr/local/bin` and `/usr/local/sbin dirs are using different permission set during generation, i.e. moving will work but it is not a root cause

Reply:
I notice that it says you are using version `1.10.0` under the `fix` job. That might be why it is succeeding for you but failing for us. The latest version is now `1.11.0`.

Reply:
@bryan-bar that is the mess by the tool author, we do really parse the content of the apt-fast script downloaded from the main branch here: https://github.com/ilikenwf/apt-fast/blob/master/apt-fast it still [pints at 1.10.0, but I guess I found the root of the issue and will come up with the solution soon-ish